examples/clock.rs

```diff
@@ -0,0 +1,223 @@
+#![no_std]
+#![no_main]
+
+/// Makes the wio_terminal appear as a USB serial port. The screen can
```

Comment:
Sorry to nitpick! Would you mind either removing or updating this comment please?